apps/console/src/extensions/i18n/resources/en-US/extensions.ts

```diff
@@ -3573,9 +3577,80 @@ export const extensions: Extensions = {
                                     "This specifies whether to notify the user via an email when password " +
                                     "recovery is successful.",
                                 label: "Notify on successful recovery"
+                            },
+                            maxResendCount: {
+                                hint: "Password recovery maximum resend count.",
+                                label: "Max resend attempts count",
```

Comment:
If this label is visible to end users, let's not use short forms. Shall we check and update the other places as well?

```suggestion
                                label: "Maximum resend attempts count",
```